src/Models/WorkTime.php

```diff
@@ -63,4 +69,24 @@ public function workTimeType(): BelongsTo
     {
         return $this->belongsTo(WorkTimeType::class);
     }
+
+    public function calculateTotalCost(): static
+    {
+        $this->total_cost = bcmul(
+            $this->user->cost_per_hour,
+            $this->total_time_ms / 1000 / 60 / 60,
+            2
+        );
+
+        if ($this->model && method_exists($this->model, 'costColumn') && $this->model->costColumn()) {
```

Comment:
$costColumn = $this->model->costColumn()
So that this does not need to be called 2 times after.